test/cpp/qps/qps_json_driver.cc

```diff
@@ -49,10 +49,107 @@ DEFINE_string(scenarios_file, "",
 DEFINE_string(scenarios_json, "",
               "JSON string containing an array of Scenario objects");
 DEFINE_bool(quit, false, "Quit the workers");
+DEFINE_string(search_param, "",
+              "The parameter, whose value is to be searched for to achieve "
+              "targeted cpu load");
+DEFINE_double(
+    initial_search_value, 0.0,
+    "initial parameter value to start the search with (i.e. lower bound)");
+DEFINE_double(targeted_cpu_load, 70.0,
+              "Targeted cpu load (unit: %, range [0,100])");
```

Comment:
I'm  not going to block this PR on this issue, but I wonder if this is really the right way to assign a target. For example, busy polling will always have a high CPU load even(actually, especially) when it's not really doing any useful work. A future PR could perhaps use a different target like median latency or 99% latency.